contracts/src/v0.8/upkeeps/ERC20BalanceMonitor.sol

```diff
@@ -0,0 +1,289 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.4;
+
+import "../ConfirmedOwner.sol";
+import "../interfaces/KeeperCompatibleInterface.sol";
+import "@openzeppelin/contracts/security/Pausable.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
```

Comment:
we recently moved to [explicit vendor dependencies in keepers](https://github.com/smartcontractkit/chainlink/blob/3a5b43e021950fcb98588d891b55641e41668091/contracts/src/v0.8/dev/keeper2_0/KeeperRegistry2_0.sol#L4-L6). Would be good to do it here too

[Style Guide](https://www.notion.so/chainlink/Solidity-Style-Guide-daa6418c56c04a45a2ff8c2655824c86#646b762840964f668bfc2ac622f4206f)